src/lib/shell/MainLoopDefault.cpp

```diff
@@ -163,22 +163,8 @@ void ProcessShellLine(intptr_t args)
 
     if (argc > 0)
     {
-        CHIP_ERROR retval = Engine::Root().ExecCommand(argc, argv);
-
-        if (retval != CHIP_NO_ERROR)
-        {
-            char errorStr[160];
-            bool errorStrFound = FormatCHIPError(errorStr, sizeof(errorStr), retval);
-            if (!errorStrFound)
-            {
-                errorStr[0] = 0;
-            }
-            streamer_printf(streamer_get(), "Error %s: %s\r\n", argv[0], errorStr);
-        }
-        else
-        {
-            streamer_printf(streamer_get(), "Done\r\n", argv[0]);
-        }
+        CHIP_ERROR error = Engine::Root().ExecCommand(argc, argv);
+        streamer_print_result(streamer_get(), error);
```

Comment:
This is no longer printing what the command was (the argv[0]), no?

Reply:
I thought it's not really useful as the user knows what he or she has just entered. Or do you think I should bring it back?

Reply:
I think it would be useful, just as a cross-check that what you think you typed and what the code saw are the same.

Reply:
OK, maybe this shouldn't be unified then as I can see that other platforms don't print it (which is more aligned with what OpenThread shell does). I'll change this PR to only affect Zephyr.